src/shared/components/species-name-parts-for-table/AssemblyAccessionIdForTable.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { type ComponentProps } from 'react';
+import classNames from 'classnames';
+
+import { AssemblyAccessionId } from '../species-name-parts';
+
+import styles from './SpeciesNamePartsForTable.module.css';
+
+type Props = ComponentProps<typeof AssemblyAccessionId>;
+
+/**
+ * In a table, assembly accession id is typically rendered with a light font.
+ */
+
+const SpeciesAssemblyIdForTable = (props: Props) => {
+  const { className: classNameFromProps } = props;
+
+  const componentClasses = classNames(
+    styles.assemblyAccessionId,
+    classNameFromProps
+  );
+
+  return <AssemblyAccessionId {...props} className={componentClasses} />;
```

Comment:
why a different file and not in this component?

Reply:
Because in tables, assembly accession id is usually rendered in a light font; but outside of tables who knows.

Reply:
No, I was saying something different I think
We could have moved `<AssemblyAccessionId/>`   as an exported component inside `AssemblyAccessionIdForTable.tsx`

Reply:
I'm still a bit lost.

The plain `AssemblyAccessionId` component just extracts the accession id from the appropriate field in the data, wraps the result in a span and returns the result.

The `AssemblyAccessionIdForTable` adds a little bit of its own opinion to this, adding a CSS class with styles to be applied by default. It wants the assembly accession id to render with a lighter font by default. I've seen this used consistently in tables, so it feels appropriate to set this style by default; but I was unsure if this is an appropriate default for other contexts. Which is why I only made `AssemblyAccessionIdForTable` add this font-weight style.

Here's a diagram illustrating the intent:

![image](https://github.com/Ensembl/ensembl-client/assets/6834224/632f4540-53ab-4321-90c2-fedea736c648)